caffe2/CMakeLists.txt

```diff
@@ -985,11 +980,11 @@ elseif(USE_CUDA)
     target_compile_definitions(torch_cuda PRIVATE USE_NCCL)
   endif()
   if(USE_UCC AND BUILD_SPLIT_CUDA)
-    target_link_libraries(torch_cuda_cpp PRIVATE __caffe2_ucc)
-    target_compile_definitions(torch_cuda_cpp PRIVATE USE_UCC)
+    target_link_libraries(torch_cuda_cpp PRIVATE ucx::ucs ucx::ucp ucc::ucc)
+    target_compile_definitions(torch_cuda_cpp PUBLIC USE_UCC)
```

Comment:
USE_NCCL is private. I wonder why USE_UCC should be public?

Reply:
This is a mistake, it should be private. Fixed.